course_catalog/tasks_helpers.py

```diff
@@ -201,7 +201,7 @@ def safe_load_json(json_string, json_file_key):
         return {}
 
 
-def digest_ocw_course(master_json, last_modified, course_instance, is_published):
+def digest_ocw_course(master_json, last_modified, course_instance, is_published, course_prefix=""):
```

Comment:
Add ` . #pylint:disable=too-many-locals` to pass tests

Reply:
Addressed in [49c6574](https://github.com/mitodl/course-catalog/pull/50/commits/49c65746f965ee9886722f59157e3aee01072eb8)